.github/workflows/deploy.yaml

```diff
@@ -462,6 +462,40 @@ jobs:
           REGISTRATION_SERVICE_VERSION: 0.0.1-SNAPSHOT
           DATASPACE_DID_HOST: ${{ needs.Deploy-Dataspace.outputs.dataspace_did_host }}
 
+      - name: 'Wait for participant registration'
+        run: |               
+          retryCount=0
+          maxRetryCount=30
+          onboardingCompleted=false
+          
+          while [ $retryCount -lt $maxRetryCount ]; do
```

Comment:
Code seems to some extent duplicated from `entrypoint.sh` consider extracting into an own file and reusing.

Reply:
I have consolidated the duplicated code.